### ISSUE TITLE
Issue in jquery bower version

### ISSUE THREAD

**Issue by hemkaran** [velesin/jasmine-jquery#299]
Hi,

I am using jasmine-jquery 2.0.7. I found two issues in specifying jquery in bower.json.

1. You have mentioned jquery instead of jQuery (which is registered as "jQuery" in bower), which overrides another jquery version instead showing a conflict message and could break the functionality.
2. You should not mention "jquery": ">=2" in bower.json, now a days jQuery 3.x is resolving this dependency which is actually breaking functionality due to point 1.

In summary, previously, "jquery": ">=2" was resolving to 2.x.x, overriding my jquery 1.11.x but was not breaking any functionality. Now,  "jquery": ">=2" is resolving to 3.x, overriding my jquery 1.11.x and also breaking my functionality.
